activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Add negative scopes for all enum values.
+
+    Example:
+    
+      class Post < ActiveRecord::Base
```

Comment:
@dhh: I think you need to add extra 2 spaces here. The example structure is broken here https://github.com/rails/rails/blob/negative-enum-scope/activerecord/CHANGELOG.md

Reply:
Also, I don't know if it matters or not, but we should inherit class `Post` from `ApplicationRecord` instead here

Reply:
I'll do the indent, but ApplicationRecord is really a Rails concern, not an Active Record concern. Maybe a fine point of difference, but it's there.